src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+function Table() {
+  return (
+    <table className="table table-bordered border-secondary">
+      <tbody>
+        <tr className="result">
+          <td colSpan={4} className="py-3">0</td>
+        </tr>
+        <tr className="numbers">
+          <td className="py-3">AC</td>
+          <td className="py-3">+/-</td>
+          <td className="py-3">%</td>
+          <td className="py-3 operator">&divide;</td>
+        </tr>
+        <tr className="numbers">
+          <td className="py-3">7</td>
+          <td className="py-3">8</td>
+          <td className="py-3">9</td>
+          <td className="py-3 operator">&times;</td>
+        </tr>
+        <tr className="numbers">
+          <td className="py-3">4</td>
+          <td className="py-3">5</td>
+          <td className="py-3">6</td>
+          <td className="py-3 operator">-</td>
+        </tr>
+        <tr className="numbers">
+          <td className="py-3">1</td>
+          <td className="py-3">2</td>
+          <td className="py-3">3</td>
+          <td className="py-3 operator">+</td>
+        </tr>
+        <tr className="numbers">
+          <td colSpan={2} className="py-3">0</td>
+          <td className="py-3">.</td>
+          <td className="py-3 operator">=</td>
+        </tr>
+      </tbody>
+    </table>
+  );
+}
+
+export default function Calculator() {
+  return (
+    <div className="container py-5">
+      <div className="table-responsive container">
+        <Table />
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] - [ ] Well done on creating a function to fill the page for good practice in REACT to having just one independent function on each file. Please 🙏🏽 divide it into two files one for Table.js and the other for calculator where you will add the corresponding function of the file.
- [ ] This is a REACT file please use it at the top of your document `import REACT from react;` for semantical reasons and in Calculator.js file after import REACT add `import Table from './Table';`